common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -7026,15 +7033,15 @@ public static void setHiveSiteLocation(URL location) {
   }
 
   public static void setHivemetastoreSiteUrl(URL location) {
```

Comment:
Why `setHivemetastoreSiteUrl`, but `getMetastoreSiteLocation`? Do you think we could refactor here?

Reply:
done